### PR TITLE
test: additional coverage for applyChangesToManuscript for all types …

### PR DESCRIPTION
**Pull request by ilya-zhuravel**
…of changes

### CODE REVIEW

File: src/model/history/add-object-change.ts [hunk 1]
```diff
@@ -21,9 +21,8 @@ export class AddObjectChange extends Change {
     return set(cloneManuscript(manuscript), this.path, [...originalSection, this.object]);
   }
 
-  public get isEmpty() { 
-    console.log('isEmpty not implimented for AddObjectChange')
-    return false; 
+  public get isEmpty() {
+    return false;
```
Comment (Cooryd) [inline src/model/history/add-object-change.ts:25]: I know this was probably removed by a linter but we should probably still flag that this functions not been implemented properly (If you could correct my miss-spelling of implemented along the way that'd be great)

File: src/model/history/delete-object-change.ts [hunk 2]
```diff
@@ -41,8 +41,7 @@ export class DeleteObjectChange extends Change {
     return set(cloneManuscript(manuscript), this.path, updatedSection);
   }
 
-  public get isEmpty() { 
-    console.log('isEmpty not implimented for AddObjectChange')
+  public get isEmpty() {
```
Comment (Cooryd) [inline src/model/history/delete-object-change.ts:44]: Same as above

File: test/unit/model/changes-utils.test.ts [hunk 3]
```diff
@@ -297,17 +297,98 @@ describe('cloneManuscript', () => {
...
+      "path": "relatedArticles.0",
+      "differences": [{"kind": "E", "path": ["href"], "lhs": "111111", "rhs": "10.7554/eLife.4849811"}]
+    }]);
+    expect(updatedManuscript.relatedArticles.length).toBe(1);
+    expect(updatedManuscript.relatedArticles[0].id).toBe(changesJSON[0].object._id);
+    expect(updatedManuscript.relatedArticles[0].articleType).toBe(changesJSON[0].object.articleType);
+    expect(updatedManuscript.relatedArticles[0].href).toBe("10.7554/eLife.4849811");
+  });
+
+  it('applies a prosemirror change on the body', () => {
```
Comment (Cooryd) [inline test/unit/model/changes-utils.test.ts:388]: This is good, shall we raise another issue for Title, Abstract, Impact Statment and Acknowledgment?

File: src/model/history/add-object-change.ts (hunk 1 above)
Comment (Cooryd) [reply src/model/history/add-object-change.ts:25]: We could potentially change these to a `console.warn` to make it clear they're warnings

Comment (ilya-zhuravel) [reply src/model/history/add-object-change.ts:25]: both AddObject and DeleteObject have this implementation in the client as well.

File: src/model/history/delete-object-change.ts (hunk 2 above)
Comment (ilya-zhuravel) [reply src/model/history/delete-object-change.ts:44]: both AddObject and DeleteObject have this implementation in the client as well.

File: test/unit/model/changes-utils.test.ts (hunk 3 above)
Comment (ilya-zhuravel) [reply test/unit/model/changes-utils.test.ts:388]: Yeah I this would be good

**Review by Cooryd — APPROVED** [review]

**Pull request merged**